lib/rules/no-useless-computed-key.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @fileoverview Rule to disallow unnecessary computed property keys on objects
+ * @author Burak Yigit Kaya
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+var MESSAGE_UNNECESSARY_COMPUTED = "Unnecessarily computed property [{{property}}] found.";
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "Disallow unnecessary computed property keys on objects",
```

Comment:
exactly as in README.md first letter of first word is lowercase

what do you think about s/on objects/in object literals/ here and in README.md and in rule doc .md file?